connector_helpscout/models/helpscout_backend/common.py

```diff
@@ -194,17 +198,24 @@ def _get_hook_values(self):
 
     # Import actions
     @api.multi
-    def action_import_mailboxes(self):
-        """Trigger an import for mailboxes on the appropriate from field."""
-        self.import_from_date('helpscout.mailbox',
-                              'import_mailboxes_from_date')
+    def action_import_conversations(self):
+        """Trigger an import for conversations on the appropriate from
+        field."""
+        self.import_from_date('helpscout.conversation',
+                              'import_conversations_from_date')
 
     @api.multi
     def action_import_customers(self):
         """Trigger an import for customers on the appropriate from field."""
         self.import_from_date('helpscout.customer',
                               'import_customers_from_date')
 
+    @api.multi
+    def action_import_mailboxes(self):
```

Comment:
We'll just implicitly import the mailboxes as dependencies to conversations, as well as provide an initial import hook. No need for cron method